src/common/CommandLineUtils.ts

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+import { Flags, SfCommand } from '@salesforce/sf-plugins-core';
+import { LoggerLevel, Messages, SfError } from '@salesforce/core';
+import { Command } from '@oclif/core';
+import { CustomOptions, OptionFlag } from '@oclif/core/interfaces';
+import { FlagsConfigType, Platform } from './Common.js';
+
+Messages.importMessagesDirectoryFromMetaUrl(import.meta.url);
+const messages = Messages.loadMessages('@salesforce/lwc-dev-mobile-core', 'common');
+
+export class CommandLineUtils {
```

Comment:
Moved this out of `Common.ts` and into its own file since `Common.ts` was getting too big and contains multiple classes